.php-cs-fixer.php

```diff
@@ -8,6 +8,7 @@
 $config = new PhpCsFixer\Config();
 return $config
     ->setRules([
```

Comment:
With risky rules we need to explicitly allow them:
```suggestion
    ->setRiskyAllowed(true)
    ->setRules([
```

Reply:
Done in https://github.com/phpmd/phpmd/pull/1138/commits/3c7b4751780f19636b825eef7c7c9f4f83978ec8